components/vector-kubearchive-log-collector/staging/stone-stg-rh01/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../base
+- loki-secret.yaml
+
+generators:
+- loki-helm-generator.yaml
+- grafana-helm-generator.yaml
+
+patches:
+- patch: |-
+    apiVersion: external-secrets.io/v1beta1
+    kind: ExternalSecret
+    metadata:
+      name: kubearchive-loki
+    spec:
+      dataFrom:
+        - extract:
+            key: integrations-output/external-resources/appsrep09ue1/konflux-internal-prod/stone-prod-p02-kube-archive-rds
```

Comment:
Remember me why it needs access to the RDS database over on production? I don't think it will be able to reach either...

Reply:
Updated the key path